src/test/completion.test.ts

```diff
@@ -69,15 +69,13 @@ suite('HTML Completion', () => {
 
 		testCompletionFor('<input t|ype="text"', {
 			items: [
-				{ label: 'type', resultText: '<input type="text"' },
```

Comment:
For the current attribute, the attribute should be in the list, e.g. to see the description or check that spelling is right